easybuild/easyconfigs/g/git-lfs/git-lfs-3.2.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'git-lfs'
+version = '3.2.0'
+
+homepage = 'https://git-lfs.github.com'
+description = """Git Large File Storage (LFS) replaces large files such as audio
+ samples, videos, datasets, and graphics with text pointers inside Git, while
+ storing the file contents on a remote server like GitHub.com"""
+
+toolchain = SYSTEM
+
+github_account = name
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['f8e6bbe043b97db8a5c16da7289e149a3fed9f4d4f11cffcc6e517c7870cd9e5']
+
+builddependencies = [('Go', '1.17.6')]
+
+files_to_copy = [(['bin/%(name)s'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/git-lfs'],
+    'dirs': [],
+}
+
+sanity_check_commands = [('git-lfs', '--version')]
+
+# set up Git LFS for the user account
+modluafooter = """
+if mode() == "load" then
+    cmdout = subprocess("git lfs install")
```

Comment:
Hmm, does this need to be done on *every* load? That seems expensive...

Also, can we add the Tcl equivalent for this?

Reply:
If it's light-weight enough, I guess this is good enough... Can we also add the `modtclfooter` equivalent though?

What happens if a user switches to a different version of either `git` or `git-lfs`? Is some cleanup needed then?

Reply:
The command `git lfs install` only modifies `.gitconfig` with the following aliases
```
[filter "lfs"]
        process = git-lfs filter-process
        required = true
        clean = git-lfs clean -- %f
        smudge = git-lfs smudge -- %f
```

So it is independent of the version of `git` or `git-lfs`.

Reply:
OK, excellent. Maybe clarify that with a comment.

Are you up for looking into the `modtclfooter` equivalent?

Reply:
done in https://github.com/easybuilders/easybuild-easyconfigs/pull/15699/commits/10cd5e823f854e1ad390b5f26cfdea58961b26d4, but I cannot test if the `modtclfooter` works :wink:

Reply:
Well, sure you can, use: `eb --from-pr 15699 --module-syntax Tcl --installpath-modules /tmp/$USER/pr15699 --force`, and then test if loaded the module works? Lmod can also consume Tcl module files...

Reply:
nice! it does work :+1: